samples/red_black_tree.cr

```diff
@@ -384,24 +384,26 @@ class RedBlackTreeRunner
 end
 
 def bench(name, n = 1)
-  t = Time.now
```

Comment:
The changes to this file are actually already done in #6454, [samples/red_black_tree.cr](https://github.com/crystal-lang/crystal/pull/6454/files#diff-f08c2a87edafd9e94150222141b6f45a).